dev-docs/bidders/zmaticoo.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: bidder
+title: zMaticoo
+description: zMaticoo Prebid Bidder Adapter
+pbjs: true
+biddercode: zmaticoo
+deals_supported: false
+tcfeu_supported: true
+tcf2_supported: true
```

Comment:
```suggestion
tcfeu_supported: false
```

This is the right setting (tcf2 does not exist) and  if you have no GVL vendor id, it's technically not supported.

Reply:
@muuki88 ok，i have gvl_id，i have change setting，please review